src/state/index.ts

```diff
@@ -21,27 +22,87 @@ import wallets from './wallets/reducer'
 
 const PERSISTED_KEYS: string[] = ['user', 'transactions', 'lists']
 
+const reducer = {
+  application,
+  user,
+  connection,
+  transactions,
+  wallets,
+  swap,
+  mint,
+  mintV3,
+  burn,
+  burnV3,
+  multicall: multicall.reducer,
+  lists,
+  logs,
+  [routingApi.reducerPath]: routingApi.reducer,
+}
+
+/* Utility type to extract state type out of a @reduxjs/toolkit Reducer type */
+type GetState<T> = T extends Reducer<infer State> ? State : never
+
+/* Utility type to mark all properties of a type as optional */
+type DeepPartial<T> = T extends object
+  ? {
+      [P in keyof T]?: DeepPartial<T[P]>
+    }
+  : T
+
+/* eslint-disable-next-line import/no-unused-modules -- reports false positive error, we use that type in other modules */
```

Comment:
This was probably a local-only error - can you remove the eslint-disable's unless the PR actually fails?